components/browser/awesomebar/src/main/java/mozilla/components/browser/awesomebar/BrowserAwesomeBar.kt

```diff
@@ -50,6 +50,7 @@ class BrowserAwesomeBar @JvmOverloads constructor(
     internal var listener: (() -> Unit)? = null
     internal var editSuggestionListener: ((String) -> Unit)? = null
     internal val styling: BrowserAwesomeBarStyling
+    var usingBottomBrowserToolbar: Boolean = false
```

Comment:
If the default is false, then wouldn't this be `usingTopBrowserToolbar`? I guess it seems weird that we have that we're not using bottom as the default and this flag is false as well.

Reply:
Maybe rename this to `positionBottom`/`positionTop` which ever way you'd like to go with it.

Reply:
I changed the naming to `customizeForBottomToolbar` so it does not imply a default position and is false by default.

Reply:
That's much nicer! 👌